src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -36,13 +36,15 @@ interface NormalizerInterface
      * @throws LogicException             Occurs when the normalizer is not called in an expected context
      * @throws ExceptionInterface         Occurs for all the other cases of errors
      */
-    public function normalize(mixed $object, string $format = null, array $context = []): array|string|int|float|bool|\ArrayObject|null;
+    public function normalize($object, string $format = null, array $context = []);
 
     /**
      * Checks whether the given class is supported for normalization by this normalizer.
      *
      * @param mixed  $data   Data to normalize
      * @param string $format The format being (de-)serialized from or into
+     *
+     * @return bool
      */
-    public function supportsNormalization(mixed $data, string $format = null): bool;
+    public function supportsNormalization($data, string $format = null);
```

Comment:
mixed stays

Reply:
Thanks for the review, sorry for making you write endless "mixed" :pray: